<?xml version="1.0"?>
<robot name="projectile">

  <link name="projectile_link">
    <visual>
      <geometry>
        <sphere radius="0.1"/>
      </geometry>
      <material name="aqua"/>
    </visual>
    <collision name="projectile_collision">
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="0.01"/>
      <inertia ixx="0.00001" ixy="0.0" ixz="0.0" iyy="0.00001" iyz="0.0" izz="0.00001"/>
    </inertial>
  </link>

  <gazebo reference="projectile_link">
    <material>Gazebo/BlueTransparent</material>

    <!-- <sensor type="contact" name="contact_sensor">
      <always_on>true</always_on>
      <update_rate>50.0</update_rate>
      <contact>
        <collision>projectile_collision</collision>
      </contact>
      <plugin name="contact_plugin" filename="libgazebo_ros_contact.so">
        <topicName>/projectile_contact</topicName>
      </plugin>
    </sensor> -->
  </gazebo>
</robot>
